x/leverage/keeper/errors.go

```diff
@@ -21,6 +21,7 @@ func nonOracleError(err error) bool {
 	if errors.IsOf(err,
 		leveragetypes.ErrInvalidOraclePrice,
 		leveragetypes.ErrNoHistoricMedians,
+		leveragetypes.ErrExpiredOraclePrice,
 		oracletypes.ErrUnknownDenom,
```

Comment:
@robert-zaremba This is where having registered error types is currently helpful.

We'd have to do this check manually by making an `OracleError` type in both the leverage and oracle modules, and then only including speciic cases, if we wanted to delete that one line registering a new error type.